post-orientation-exercises/express-static/02-templating-pug.md

```diff
@@ -4,12 +4,57 @@
 
 #### What is a templating engine?
 
-A template engine enables you to use static template files in your application. At runtime, the template engine first parses the template. This allows embedded JavaScript expressions and variables to be evaluated and then transformed into an HTML file which is sent to the client. This approach creates the ability to pipe data from the server to the template, so it can then be parsed and sent to the client.
+A template engine enables you to use static template files in your application. At runtime, the template engine first parses the template. This allows embedded JavaScript expressions and variables to be evaluated and then transformed into an HTML file which is sent to the client. This approach creates the ability to pipe data from the server to the template, so it can then be parsed and sent to the client. If that sounds familiar, it should. You have worked with templates before with Handlebars, and in the Anugular templating system.
 
 #### Pug
 
 Pug is a clean, whitespace sensitive syntax for writing html. We will be using the Pug templating engine with our Express server. Check out the [PugJs docs](https://pugjs.org/api/getting-started.html) for more information.
 
+BTW, Pug used to be called Jade. You will see references to Jade all over the place on the interwebs, since it was called that for a number of years.
+
+Pug and Handlebars are very similar in their goals and end results. You're still going to bind javascript values to templates to create views to render in a broswer. But the implementation is slightly different. the first thing you'll notice is the syntax. Pug uses HTML tags, but in a whitespace-dependent syntax that does away with angle brackets (`< >`) and closing tags. Proper indentation is _required_. Without properly indented
```

Comment:
typo: broswer -> browser

Reply:
typo: 'the first thing' -> 'The first thing'